recipe/build.sh

```diff
@@ -1,34 +1,17 @@
-cd src/cxx
-
-autoconf
-
-export CPATH="${PREFIX}/include"
 
-./configure --prefix=$PREFIX --enable-noisy-make
+cd src/common_libraries/libsharp
```

Comment:
I didn't realize this build vendored libsharp. We have a version of libsharp that we ship standalone: https://github.com/conda-forge/libsharp-feedstock

Can we link to that instead?

Reply:
I tried to compile it using the libsharp standalone, but it runs into an issue since it cannot find `sharp_execute` in the `libsharp.a` library:
```
checking for libsharp... yes
checking for sharp_execute in -lsharp... no
configure: error: could not find the libsharp library
```

Reply:
Is libsharp statically linked or dynamically?

Reply:
It's statically linked in the `libsharp` standalone package.

Reply:
OK then all we need to change is to make sure the libsharp license is included in the license files in the meta.yaml if it is not already there.

Reply:
It seems that the license for `libsharp` is exactly the same as for `healpix_cxx`: https://github.com/Libsharp/libsharp/blob/master/COPYING
and is exactly the same name. Is possible to include both licenses?

Reply:
you can rename the file in the build and then include it.

Reply:
Okay done.